descriptors/ModuleDescriptor-template.json

```diff
@@ -93,7 +93,7 @@
     },
     {
       "id": "circulation-logs",
-      "version": "1.1",
+      "version": "1.2",
```

Comment:
The only this interface version can be increased. The last ones should be rolled back because they aren't affected by MODAUD-79.

Reply:
Fixed, thank you.